Sources/PackageCollections/Model/Collection.swift

```diff
@@ -66,8 +71,8 @@ extension PackageCollectionsModel {
             packages: [Package],
             createdAt: Date,
             createdBy: Author?,
-            lastProcessedAt: Date = Date(),
-            isSigned: Bool
+            signature: SignatureData?,
+            lastProcessedAt: Date = Date()
```

Comment:
Does this need a compatibility wrapper callback or are there no clients using this yet?

Reply:
The initializer is not exposed outside of this module so we should be ok.